docker-compose.yml

```diff
@@ -9,4 +9,11 @@ services:
       - "4200:4200"
     container_name: edlinker-front-end
     volumes:
-      - .:/app
\ No newline at end of file
+      - .:/app
+
+  mock-json:
+    image: 'ajoelpod/mock-json-server'
```

Comment:
я хз що це за докерфайл тому краще зроби свій
```Dockerfile
FROM node:14.15.5-alpine3.10

RUN npm install -g json-server
CMD ["json-server", "--watch", "/usr/src/app/data.json"]
```

в докер-компос файлі

```yml
  mock_api:
    build:
      context: .
      dockerfile: Dockerfile.mock-api
    volumes:
      - ./api/db.json:/usr/src/app/data.json
    ports:
      - 3000:3000
```

Reply:
і ти походу юзаєш якусь ноунейм лібу для цього

Reply:
https://hub.docker.com/r/vimagick/json-server

Reply:
Яка там версія ноди в цьому імеджі? Як ти будеш її оновлювати? Краще зроби свій. Це не постгрес підняти в руками

Reply:
Error: **### The webpage at http://localhost:3000/posts might be temporarily down or it may have moved permanently to a new web address**

![image](https://user-images.githubusercontent.com/70710636/109795149-45b12600-7c1f-11eb-93a0-5f99a6db4c59.png)
![image](https://user-images.githubusercontent.com/70710636/109795207-53ff4200-7c1f-11eb-8113-3588f4fb317e.png)

Reply:
@RostyslavHavuka Покажи що дає тобі джейсон сервер при запуску

Reply:
@TArch64 маєш на увазі ось це?
![image](https://user-images.githubusercontent.com/70710636/109803531-d42aa500-7c29-11eb-90b3-7912d0d776ae.png)
![image](https://user-images.githubusercontent.com/70710636/109803596-e9073880-7c29-11eb-88b1-91fad9a965a1.png)

Reply:
@RostyslavHavuka У мене все працює. Ти точно перебілдив імеджі?
<img width="681" alt="image" src="https://user-images.githubusercontent.com/29049264/109804400-df320500-7c2a-11eb-9480-ebdf47c111ff.png">
```Dockerfile
FROM node:14.15.5-alpine3.10

RUN npm install -g json-server
CMD ["json-server", "--host", "0.0.0.0", "--watch", "/usr/src/app/data.json"]
```
```yml
version: "3.3"
services:
  mock_api:
    build:
      context: .
      dockerfile: Dockerfile.mock-api
    volumes:
      - ./api/db.json:/usr/src/app/data.json
    ports:
      - 3000:3000
```

Reply:
@TArch64, все ок соррі за пінг) 
Я не перебілдив просто.